### PR TITLE
Removed the use of config, used more of a blackbox exporter's (https:…

### PR DESCRIPTION
**Pull request by mishas**
…//github.com/prometheus/blackbox_exporter) approach

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
-# prometheus_amqp_proxy
-Proxy for feeding prometheus scraper from AMQP RPC source
+# Prometheus AMQP proxy
+
+Proxy for feeding prometheus scraper from AMQP RPC sources.
+
+## Building and running
+
+### Build and run using [bazel](http://baze.io/)
```
Comment (nadirizr) [inline README.md]: s/baze.io/bazel.io/

Comment (nadirizr) [inline README.md]: And also consider going straight to the installation page.

File: proxy/proxy.go [hunk 2]
```diff
@@ -3,18 +3,27 @@ package main
...
 
-	"github.com/mishas/prometheus_amqp_proxy/proxy/config"
 	"github.com/mishas/prometheus_amqp_proxy/proxy/rpc"
 )
 
+var (
+	amqpURL      = flag.String("amqp_url", "", "URL of the AMQP server to dial")
+	amqpExchange = flag.String("amqp_exchange", "", "Name of the AMQP exchange to use")
+
+	certsDir   = flag.String("certs_dir", "", "Directory of certs for TLS connection to AMQP, or empty for non-TLS connection")
```
Comment (nadirizr) [inline proxy/proxy.go]: Maybe you should document the names of the files within the directory that are expected.

File: README.md (hunk 1 above)
Comment (mishas) [inline README.md]: Done.

File: proxy/proxy.go (hunk 2 above)
Comment (mishas) [inline proxy/proxy.go]: Done.

**Pull request merged**